packages/glimmer-runtime/tests/attributes-test.ts

```diff
@@ -232,10 +232,10 @@ test("a[href] can be removed by setting to `null`", () => {
 
   rerender({ foo: null });
 
-  equalTokens(root, '<a></a>');
+  equalTokens(root, '<a href="null"></a>');
```

Comment:
The test expected value was correct previously (though I realize that this failure cascaded out of a fix for the other things).

If we can't fix this, I understand (I am definitely in favor of staying as close to what was shipped in 2.4 and 2.8 as possible), but it really does seem like a bug to me.